docs/DefaultApi.md

```diff
@@ -1820,6 +1826,490 @@ Name | Type | Description  | Notes
 
 [[Back to top]](#) [[Back to API list]](../README.md#documentation-for-api-endpoints) [[Back to Model list]](../README.md#documentation-for-models) [[Back to README]](../README.md)
 
+# **get_ten_dlc_sms_brand**
+> SMSTenDLCBrand get_ten_dlc_sms_brand(account_id, brand_id)
+
+Get a 10DLC SMS Brand
+
+### Example
+
+* Basic Authentication (fc):
+
+```python
+import time
+import freeclimb
+from freeclimb.api import default_api
+from freeclimb.model.sms_ten_dlc_brand import SMSTenDLCBrand
+from pprint import pprint
+# Defining the host is optional and defaults to https://www.freeclimb.com/apiserver
+# See configuration.py for a list of all supported configuration parameters.
+configuration = freeclimb.Configuration(
+    host = "https://www.freeclimb.com/apiserver"
+)
+
+# The client must configure the authentication and authorization parameters
+# in accordance with the API server security policy.
+# Examples for each auth method are provided below, use the example that
+# satisfies your auth use case.
+
+# Configure HTTP basic authorization: fc
+configuration = freeclimb.Configuration(
+    username = 'ACCOUNT_ID',
+    password = 'API_KEY'
+)
+
+# Enter a context with an instance of the API client
+with freeclimb.ApiClient(configuration) as api_client:
+    # Create an instance of the API class
+    api_instance = default_api.DefaultApi(api_client)
+    account_id = "accountId_example" # str | ID of the account
+    brand_id = "brandId_example" # str | String that uniquely identifies this brand resource.
+
+    # example passing only required values which don't have defaults set
+    try:
+        # Get a 10DLC SMS Brand
+        api_response = api_instance.get_ten_dlc_sms_brand(account_id, brand_id)
```

Comment:
the account id is automatically loaded from the initial configuration, should not be needed here. Documentation needs to be updated